mintlify/uma-connect/developer-guides/reconcilation.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: "Reconciliation"
```

Comment:
What would be in this page? Remember that SoFi's current setup is temporary and won't be the intended usage flow of UMAaaS (which includes real time funding).

Reply:
If I were a fintech, I'd probably what to verify that the transactions I saw in a day matched the tx on the platform.  There may be cases where maybe there's issues with webhook handling.  

This page will provide instructions on how they can handle reconciliation via GET /transactions and filter by dates / statuses